lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
@blankenberg Hrmm, ok, I see that argument.  So, https://github.com/galaxyproject/galaxy/commit/4cd2af860ffc31ee574e6f81c941840ae603efe5 was in error?